scale/scheduler/test/database/test_updater.py

```diff
@@ -135,6 +138,22 @@ def test_update_batch_fields(self):
         batch_test_utils.create_batch_job(batch=batch_2, job=job_3)
         batch_test_utils.create_batch_job(batch=batch_2, job=job_4)
 
+        # This batch tests an old batch that never created any recipes
+        time_rev_1 = now()
+        time_rev_2 = time_rev_1 + timedelta(minutes=1)
+        time_batch = time_rev_2 + timedelta(minutes=1)
+        time_rev_3 = time_batch + timedelta(minutes=1)
+        recipe_type_3 = recipe_test_utils.create_recipe_type()
+        recipe_test_utils.edit_recipe_type(recipe_type_3, recipe_type_3.definition)
+        recipe_test_utils.edit_recipe_type(recipe_type_3, recipe_type_3.definition)
```

Comment:
duplicate line?

Reply:
Actually no. I need three revisions of the recipe type so I create it and then edit it twice.